jerry-core/jerryscript.h

```diff
@@ -389,6 +389,11 @@ size_t jerry_parse_and_save_literals (const jerry_char_t *source_p, size_t sourc
                                       uint32_t *buffer_p, size_t buffer_size, bool is_c_format);
 
 /**
+ * External context functions.
+ */
+size_t jerry_get_context_size (void);
+size_t jerry_get_lcache_size (size_t row_count, size_t row_length);
```

Comment:
Can we decide that there only will be 2 "globals" for each instance, one is heap, and the other is context. In the future, if we want to change/extend the "globals", we just change/add the items in context.